source/class/qx/tool/utils/QooxdooVersions.js

```diff
@@ -0,0 +1,190 @@
+/* ************************************************************************
+
+   qooxdoo - the new era of web development
+
+   http://qooxdoo.org
+
+   Copyright:
+     2023 Zenesis Limited https://www.zenesis.com
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+     See the LICENSE file in the project's top-level directory for details.
+
+   Authors:
+     * John Spackman (@johnspackman)
+
+************************************************************************ */
+
+const semver = require("semver");
+const path = require("upath");
+const fs = qx.tool.utils.Promisify.fs;
+
+qx.Class.define("qx.tool.utils.QooxdooVersions", {
+  extend: qx.core.Object,
+
+  construct() {
+    super();
+  },
```

Comment:
Superfluous constructor, since all it does it call the superclass constructor, but not wrong.